src/main/java/io/github/amerebagatelle/fabricskyboxes/mixin/skybox/SunSkyColorMixin.java

```diff
@@ -16,10 +18,12 @@ public class SunSkyColorMixin {
             constant = @Constant(intValue = 4, ordinal = 0)
     )
     private static int renderSkyColor(int original) {
-        if (SkyboxManager.renderSunriseAndSet) return original;
-        else {
-            SkyboxManager.renderSunriseAndSet = true;
-            return Integer.MAX_VALUE;
+        Skybox skybox = SkyboxManager.getInstance().getCurrentSkybox();
+        if (skybox instanceof FSBSkybox fsbSkybox) {
```

Comment:
Same as above here.  When does this check benefit?

Reply:
Response mentioned above.